index.md

```diff
@@ -34,8 +34,9 @@ Gerrit, Launchpad, Jira, mailing lists, Confluence, Discourse, Slack, Jenkins,
 Meetup, Mediawiki, Phabricator, Redmine, StackOverflow, Telegram, and others.
 
 Let´s now visit the main functionalities provided by GrimoireLab. The section
-[Components](/components) shows how the different components in GrimoireLab are
-combined to provide these functionalities.
+[Components]({{ site.baseurl }}{% link docs/components/components.md %}) shows
```

Comment:
we should do the same every other link present in the doc from now on ?

Reply:
Some links don't work because the _baseurl_ won't be included, not in the case when we are working locally. I have now configured _baseurl_ with the local setup too. I think it is because the content is present in different folders, so the link doesn't work.

If the link doesn't work, try adding it in this format.